pkgs/by-name/sd/SDL_compat/package.nix

```diff
@@ -1,17 +1,20 @@
 {
   lib,
   SDL2,
+  sdl2-compat,
   cmake,
   darwin,
   fetchFromGitHub,
   libGLU,
   libiconv,
+  libX11,
   mesa,
   pkg-config,
   stdenv,
   # Boolean flags
   libGLSupported ? lib.elem stdenv.hostPlatform.system mesa.meta.platforms,
   openglSupport ? libGLSupported,
+  useSDL3 ? true,
```

Comment:
Do we actually need this option?

Reply:
i believe not; `.override { sdl2-compat = SDL2; }` is simple enough if needed occasionally, and could always be an alias in `all-packages.nix` if it ever becomes a recurring pattern.

Reply:
Same vibe. Lets just drop it.